docs/source/integrations/middleware.mdx

```diff
@@ -396,6 +411,10 @@ You _must_ `await server.start()` before calling `server.applyMiddleware`. You c
 * `onHealthCheck`
 * `disableHealthCheck`
 
+`applyMiddleware` also accepts the following options:
+
+* `routeGet` - get specific route options
```

Comment:
in the description here, capitalize GET and POST — so it's clear that we're talking about the methods rather than saying that this is how you get route options.  Or perhaps "route options for GET requests"?